x-pack/plugins/snapshot_restore/public/application/components/policy_form/steps/step_review.tsx

```diff
@@ -220,13 +221,16 @@ export const PolicyStepReview: React.FunctionComponent<StepProps> = ({
               {includeGlobalState === false ? (
                 <FormattedMessage
                   id="xpack.snapshotRestore.policyForm.stepReview.summaryTab.includeGlobalStateFalseLabel"
-                  defaultMessage="No"
+                  defaultMessage="No, without any feature states"
```

Comment:
Isn't the feature state info shown separately?

Reply:
Since they are tightly coupled in the UI we also thought that it made sense to keep them together in the review step